ESCPOS_NET/Printers/NetworkPrinter.cs

```diff
@@ -35,13 +34,10 @@ private void Connect()
             _socket = new Socket(_endPoint.AddressFamily, SocketType.Stream, ProtocolType.Tcp);
             _socket.Connect(_endPoint);
             _sockStream = new NetworkStream(_socket);
-            _writer = new BinaryWriter(_sockStream);
-            _reader = new BinaryReader(_sockStream);
-        }
 
-        ~NetworkPrinter()
-        {
-            Dispose();
+            // Need to review the paramaters set here
```

Comment:
do you have some concerns about these params?  what encoding was it using before, I would think the default would be UTF8

Reply:
You're right, this is the default, I think I was checking various settings, because of connection closing/time out issue. Happy to revert these.